sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClient.cs

```diff
@@ -145,6 +145,15 @@ public ResourceGroupOperations GetResourceGroupOperations(ResourceGroupResourceI
             return new ResourceGroupOperations(new SubscriptionOperations(new ClientContext(ClientOptions, Credential, BaseUri, Pipeline), id.SubscriptionId), id.ResourceGroupName);
         }
 
+        /// <summary>
+        /// Gets the tags operations under this subscription.
+        /// </summary>
+        /// <returns> The tags operations. </returns>
+        public virtual PreDefinedTagsOperations GetTagOperations()
```

Comment:
name of the method should be Get[Resource]Operations

Reply:
Done